lib/services/logs.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+import 'package:flutter/widgets.dart';
+import 'package:mobile_nebula/services/result.dart';
+
+class LogsNotFoundException implements Exception {
+  String error() => 'No logs found. Has the site been started?';
```

Comment:
@nbrownus, you good with this wording?

Reply:
The question feels weird to me, unless there was some catastrophic error during a site start then there would be at least 1 log line in a file to read. 

"Logs will be available after starting the site for the first time" or something feels more helpful